projects/github-actions/repo-gardening/CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.1.0] - 2021-03-31
+### Added
+- Add a new task to notify Editorial team when we want their feedback.
+
+### Changed
+- Automatically add "JS Package" PR label.
+- Expand list of labels to clean up after a PR has been merged.
+- Mark parameters that are not used by all tasks as optional.
+
+### Fixed
+- handle production files for external use
+
 ## 1.0.0 - 2020-07-07
 
 - Initial release
+
+[1.1.0-beta]: https://github.com/Automattic/action-repo-gardening/compare/v1.0.0...v1.1.0
```

Comment:
How'd this happen? Is it a bug in changelogger, or did you accidentally release 1.1.0-beta then manually edit?
```suggestion
[1.1.0]: https://github.com/Automattic/action-repo-gardening/compare/v1.0.0...v1.1.0
```

Reply:
Yes, that's exactly that, my bad. I used `-b` without meaning to.